docs/connectors.md

```diff
@@ -18,12 +14,65 @@ Give the issue a `+1` if you really need that connector. The upvote will help th
 particular connector. If you find that an issue hasn't been created for your data store, please create a new issue in
 the Conduit repo.
 
-## Connectors
+## More about Connectors
+
+### Built-in connectors
+
+To help developers bootstrap pipelines much more quickly,
+Conduit ships with several built-in connectors by default.
+This includes the Postgres, File, Random Data Generator, Kafka and Amazon S3 connectors.
+
+If you want to create your own Conduit binary with a different set of built-in connectors, you can:
+
+- Find an [existing standalone connector](#the-list) that you would like to build-in
+or you can [create your own with the Conduit Connector SDK](https://github.com/ConduitIO/conduit-connector-sdk)
+- Import the repository defining the connector
+into the [builtin registry](https://github.com/ConduitIO/conduit/blob/main/pkg/plugin/builtin/registry.go)
+and add a new key to the `DispenserFactory`:
+
+```go
+package builtin
+
+import (
+  // ...
+  file "github.com/conduitio/conduit-connector-file"
+  // ...
+  myNewConnector "github.com/foo/conduit-connector-new"
```

Comment:
you'd need to `go get "github.com/foo/conduit-connector-new"` first, for this line to not fail, so maybe we should add this as a step?

Reply:
Good catch, and thank you, I can add this step as well!